test/runtests.jl

```diff
@@ -212,16 +212,19 @@ mkpath(TEST_EXAMPLES_DIR)
     @testset "get scalar cell data" begin
       cell_data_raw = get_data(get_cell_data(vtk)[cell_data_name])
       cell_data_reshaped = reshape(cell_data_raw, ((Nx-1), (Ny-1), (Nz-1)))
-      @test cell_data_reshaped == cell_scalar_field
+      cell_data_reshaped1 = get_data_reshaped(get_cell_data(vtk)[cell_data_name], cell_data=true)
+      
+      @test cell_data_reshaped == cell_scalar_field 
     end
 
     @testset "get scalar point data" begin
       point_data_raw = get_data(get_point_data(vtk)[point_data_name])
       point_data_reshaped = reshape(point_data_raw, (Nx, Ny, Nz))
-      @test point_data_reshaped == point_scalar_field
+      point_data_reshaped1 =  get_data_reshaped(get_point_data(vtk)[point_data_name])
+      
+      @test point_data_reshaped == point_scalar_field == point_data_reshaped1
```

Comment:
```suggestion
      @test point_data_reshaped == point_scalar_field
      @test point_data_reshaped1 == point_scalar_field
```
I'd make two tests out of it such that we get the line number in case only one of them fails